src/components/date-range/date-range.d.ts

```diff
@@ -74,13 +85,15 @@ export interface DateRangeProps extends MarginProps {
    */
   startInfo?: boolean | string;
   /** An array containing the value of startDate and endDate */
-  value?: string[];
+  value: string[];
   /** When true, validation icons will be placed on labels instead of being placed on the inputs */
   validationOnLabel?: boolean;
   /** Overrides the default tooltip position */
   tooltipPosition?: "top" | "bottom" | "left" | "right";
 }
 
+declare const DateRangeContext: React.Context<DateRangeContextProps>;
```

Comment:
This should be in a `date-range-context.d.ts` file as that's where `DateRangeContext` is defined